source/standards/git.html.md.erb

```diff
@@ -18,6 +18,16 @@ developers on your project. This includes:
 * any future developers (including yourself) who want to see why a change was
   made
 
+Taking the time to write a good commit message is important for a few reasons.
+
+It's being kinder to our future selves. Like an
+[architecture decision record][adr], capturing context around a change allows
+people to understand why a particular implementation decision was made.
+ 
+It's an advertisement to work with you. Its demonstrates that you are
+experienced enough to care about future maintainers. It's a palpable form of
```

Comment:
"experienced enough" doesn't sound right - you can have lots of experience and not care, or you can be a new junior and get it straight away.  Maybe "It demonstrates that you care about future maintainers" is enough?

Reply:
Agreed - will edit. Thanks :)